www-apache/modsecurity-crs/modsecurity-crs-3.3.4.ebuild

```
@@ -0,0 +1,42 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+DESCRIPTION="OWASP ModSecurity Core Rule Set"
+HOMEPAGE="https://coreruleset.org/"
+SRC_URI="https://github.com/coreruleset/coreruleset/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/coreruleset-${PV}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+apache2 nginx"
+
+RDEPEND="
+	apache2? ( >=www-apache/mod_security-2.9.6 )
+	nginx? ( >=dev-libs/modsecurity-3.0.8 )
```

```suggestion
	apache2? ( >=www-apache/mod_security-2.9.6 )
	!apache? ( >=dev-libs/modsecurity-3.0.8 )
```
in that case would this make sense?

---

I would find it more suprising as seeing apache2 / nginx use flags.